framework/test/jest/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+	"extends": ["../../../.eslintrc.json", "plugin:jest/recommended"],
+	"plugins": ["jest"],
+	"env": {
+		"jest/globals": true
+	},
+	"rules": {
+		"import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
+
+		// Disable mocha related lisk-base rules
```

Comment:
We need to find a better way to support both test frameworks on the eslint base repository.

Reply:
I agree but not in this PR.

Reply:
Please, raise an issue in that repo and link it to Core.

Reply:
I'm not sure if we should create it in this repository, I think it's more lisk-style related or `eslint-config-lisk-base` would be better. I think we should have an STM before creating an issue and come up with a strategy.